dicodile/update_z/dicod.py

```diff
@@ -199,9 +199,9 @@ def reconstruct_pobj(X, D, reg, _log, t_init, t_reduce, n_workers,
 
 
 def _spawn_workers(n_workers, hostfile):
-    comm = get_reusable_workers(n_workers, hostfile=hostfile)
-    send_command_to_reusable_workers(constants.TAG_WORKER_RUN_DICOD)
-    return comm
+    workers = get_reusable_workers(n_workers, hostfile=hostfile)
```

Comment:
Maybe better to use directly `Workers` here no?
```suggestion
    workers = Workers(n_workers, hostfile=hostfile)
```